wdl/src/main/scala/wdl/expression/WdlStandardLibraryFunctions.scala

```diff
@@ -98,15 +97,17 @@ trait WdlStandardLibraryFunctions extends WdlFunctions[WomValue] {
       womString <- WomStringType.coerceRawValue(pattern)
       patternString = womString.valueString
       filePaths <- Try(globHelper(patternString))
-    } yield WomArray(WomArrayType(WomSingleFileType), filePaths.map(WomSingleFile))
+    } yield WomArray(WomArrayType(WomSingleFileType), filePaths.map(WomSingleFile(_)))
   }
 
   def basename(params: Seq[Try[WomValue]]): Try[WomString] = {
 
     val arguments: Try[(WomValue, WomValue)] = params.toList match {
       case Success(f) :: Nil => Success((f, WomString("")))
       case Success(f) :: Success(s) :: Nil => Success((f, s))
-      case s if s.size > 2 || s.size < 1 => Failure(new IllegalArgumentException(s"Bad number of arguments to basename(filename, suffixToStrip = ''): ${params.size}"))
+      case s if s.lengthCompare(2) > 0 || s.lengthCompare(1) < 0 =>
```

Comment:
ToL: it might be "better" scala but IMO their new syntax is way less readable

Reply:
> new syntax is way less readable

That should be scala's official motto 😉